src/icons/ChevronDownIcon.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
```

Comment:
Just curious-why are these SVGs implemented as React components rather than standalone .svg files?

Reply:
Ivan, I can see one of the reason is that one wants to be able to attach custom props e.g. width, height, onclick to the component.